src/Travis/Client.php

```diff
@@ -59,6 +65,16 @@ public function fetchRepository($slug)
         return $repository;
     }
 
+    public function restartBuild($build) {
```

Comment:
Seems like you might want to move `Client::restartBuild($build)`, to maybe  `Build::restart()` that uses $build->id internally.

That way you could do something like...
```php
$repository = $client->fetchRepository('org/repo-name');
$repsotory->getLastBuild()->restart();
```